app/views/layouts/_flash_messages.html.slim

```diff
@@ -1,4 +1,9 @@
 - flash.each do |variant, message|
   - next unless variant.in? t("banners").keys.map(&:to_s)
-  = govuk_notification_banner title_text: t("banners.#{variant}"), classes: "govuk-notification-banner--#{variant} govuk-!-margin-top-3 govuk-!-margin-bottom-0" do |notification_banner|
-    - notification_banner.with_heading(text: message)
+  - if variant.to_s == "success" && message.include?(t("subscriptions.create.success"))
```

Comment:
We have a requirement to trigger a JS call when the subscription confirmation page gets displayed. And since the page is just the subscriptions index with a flash success message... this is the only way I found to identify & trigger that event: "If the flash message is a subscription confirmation, include this marketing tracking element"